addons/mail/static/src/scss/mail_activity.scss

```diff
@@ -23,6 +23,20 @@
     }
 }
 
+/* next_activities badge widget */
+.o_field_widget.o_field_badge[name="next_activities"] {
+    max-width: var(--Tag-max-width, 200px);
+
+    .badge {
+        /* we can also use `@ extend .o_tag.o_tag_color_0;` for tag style instead of this hard-coded,
+           but it will give check style warning in runbot, we'll see in validation if it's okay or not. */
+        background-color: #e6dddd;
+        font-size: small;
+        font-weight: 400;
+        line-height: 1.5;
+        }
+    }
+
```

Comment:
Yes, as we need to show the value in Tags Style, as I mention we use `@extend .o_tag.o_tag_color_0` method also, but that was giving check style error. (we can also do this by making a custom js widget but that would be too much code, IMO its better to use `badge` and overwrite it in css)

Reply:
can we use  `@include`?
can you please check it let me know

Reply:
we can use `@include` if `o_tag` or `o_tag_color` is a `mixin, but its not.

Reply:
Hello @yagp-odoo 
Had another though can we use some sort of decorations in the field
using decoration- can you please check that
Thanks :smiley:

Reply:
decorations does not have that specific color we need, and also while using decorations with badge it will also not have same style as we saw in tags (I.e., font size, weight, line height... will be different, if we just use decorations that is)